README.md

```diff
@@ -22,10 +22,11 @@ Disclaimer - this software has no association to `Corsair Gaming, Inc` , is prov
 		<add key="HeadsetName" value="Headphones"/> //Use your headphone's name
 		<add key="SpeakersName" value="Speakers"/> //if you have more than one device named "Speakers"(for example if you use steam remote play you might heve a device named "Steam Streaming Speakers", which will too show up in the command output), then use the full name of the device as reported by get-audiodevice -list | where Type -eq 'Playback' in powershell
 		<add key="Autorun" value="true"/>
-		<add key="ConnectedRegexOverride" value="\d{4}\-\d\d\-\d\d\s(?:\d\d\:)*\d\d.\d+ I cue\.dev\.connection_proxy: .?(\w*).? Disconnected -> (?:Wireless|Direct)"/> //this is a "device connected" regex override for iCUE 5
-		<add key="DisconnectedRegexOverride" value="\d{4}\-\d\d\-\d\d\s(?:\d\d\:)*\d\d.\d+ I cue\.dev\.connection_proxy: .?(\w*).? (?:Wireless|Direct) -> Disconnected"/> //this is a "device disconnected" regex override for iCUE 5
+		<add key="ConnectedRegexOverride" value='\d{4}-\d\d-\d\d\s(?:\d\d:?)+\.\d+\sI\scue\.dev\.connection_(?:controller)|(?:proxy):\s"(?:\w|\s|\d)+"\sDisconnected\s->\s(?:Wireless)|(?:Direct)'/> //this is a "device connected" regex override for iCUE5
+		<add key="DisconnectedRegexOverride" value='\d{4}-\d\d-\d\d\s(?:\d\d:?)+\.\d+\sI\scue\.dev\.connection_(?:controller)|(?:proxy):\s"(?:\w|\s|\d)+"\s(?:Wireless)|(?:Direct)\s->\sDisconnected'/> //this is a "device disconnected" regex override for iCUE5
```

Comment:
The quotes were initially just dot-question (`.?`) as I was initially unsure of how to write it, moreover I have this newer config on my instance and it works just fine

But if standardisation is the name of the game, then sure I can revert them to `.?` with " wrap

Reply:
Hmm i'm just not sure your change works for the previous versions of iCue or not, since there were no quotes around that group. But if you say it works I don't mind merging it in.